string[] joint_name
float64[] delta_position
